staking/types/validator.go

```diff
@@ -19,8 +19,8 @@ import (
 
 // Define validator staking related const
 const (
-	MaxNameLength            = 70
-	MaxIdentityLength        = 3000
+	MaxNameLength            = 140
```

Comment:
max lengths are changed now? We need to update cli too.

Reply:
Yea, please. take a look at the test plan.